test/publish.test.js

```diff
@@ -102,12 +103,13 @@ test('publish (DMs: myFeedId + feedId)', async t => {
   const content = {
     type: 'announce',
     text: 'summer has arrived in wellington!',
+    // TODO: failing here because the first recp isn't a groupId. the old code just published anyway. do we still want to do that? i think the spec might allow it but we disallow it in tribes2
```

Comment:
@mixmix i also commented about this in some other place. what do you think?

Reply:
`recps[0]` can be a `feedId`